lib/kbsecret/session.rb

```diff
@@ -80,6 +80,16 @@ def add_record(type, label, *args)
       record.sync!
     end
 
+    # Load an existing record
+    # @param record [Record] an instance of a record
+    # @return [void]
+    def import_record(record)
+      klass = record.class
```

Comment:
Let's also throw some kind of exception if the user tries to import a record into the same session:

```ruby
raise Exceptions::RecordDuplicationError.new(self, record.session) if self == record.session
klass = record.class
# ...
```

Reply:
(This allows us to propagate the error back up to the user).